deploy/docker-compose.yml

```diff
@@ -15,11 +15,13 @@ services:
     volumes:
       - ./nginx/default.conf.template:/etc/nginx/templates/default.conf.template:ro
     environment:
-      - S3_PUBLIC_URL=${S3_PUBLIC_URL:-http://localhost/s3/}
-      - S3_LOCAL_URL=${S3_lOCAL_URL:-http://minio:9000}
+      - FUZON_PUBLIC_URL=${FUZON_PUBLIC_URL:-http://localhost/fuzon}
       - HTSGET_PUBLIC_URL=${HTSGET_PUBLIC_URL:-http://localhost/htsget}
+      - S3_PUBLIC_URL=${S3_PUBLIC_URL:-http://localhost/s3/}
+      - FUZON_LOCAL_URL=${FUZON_LOCAL_URL:-http://fuzon:8080}
       - HTSGET_LOCAL_URL=${HTSGET_LOCAL_URL:-http://htsget:8080}
```

Comment:
**question(clarification)**: from what I understand both fuzon and htsget will use the same port for each service to run. Is this fine in general? Networking is by far my weakest point, so I hope the question is not too basic.

Reply:
sure, that can be confusing; they use the same port, but on different containers:
when on localhost:
* fuzon is `http://fuzon:8080`
* htsget is `http://htsget:8080`

However, when accessing from outside the machine, everything is accessed through a reverse proxy (see [the nginx config](https://github.com/sdsc-ordes/modos-api/blob/feat/fuzon-codes/deploy/nginx/default.conf.template#L37-L48)) which redirects incoming requests to local services as follows:
* http://<server>:80/fuzon -> fuzon:8080
* http://<server>:80/htsget -> htsget:8080